d3/query-graphs.js

```diff
@@ -592,6 +594,47 @@ function drawQueryTree(target, treeData) {
         update(d);
     }
 
+    // Dash tween to make the highlighted edges animate from start node to end node
+    var tweenDash = function() {
+        var l = this.getTotalLength();
+        var i = d3.interpolateString("0," + l, l + "," + l);
+        return function(t) {
+            return i(t);
+        };
+    };
+
+    // Transition used to highlight edges on mouseover
+    var edgeTransitionIn = function(path) {
+        path.transition()
+            .duration(duration)
+            .attr("opacity", 1)
+            .attrTween("stroke-dasharray", tweenDash)
+            .attr("d", diagonal);
+    };
+
+    // Transition to unhighlight edges on mouseout
+    var edgeTransitionOut = function(path) {
+        path.transition()
+            .duration(duration)
+            .attr("opacity", 0)
+            .attr("d", diagonal);
+    };
+
+    // Handler builder for mouseover and mouseout
+    var mouseHandler = function(transition) {
```

Comment:
this function is crosslink specific, but `mouseHandler` suggests a more general scope of this function.
Maybe rename to something like `crosslinkHighlightHandler`?
And also update the comment accordingly

Reply:
Agreed.